tools/min-base-version

```diff
@@ -28,14 +28,14 @@ from glob import glob
 proj_dir = os.path.dirname(os.path.dirname(os.path.realpath(sys.argv[0])))
 
 max_version = '0'
-for manifest in glob(os.path.join(proj_dir, 'pkg/*/manifest.json')):
+for manifest in glob(os.path.join(proj_dir, 'pkg/*/manifest.json.in')):
```

Comment:
Right, let's handle that in a separate PR. Right now the dependency calculation is completely broken, so one way or another we need to fix this for 133.